src/imitation/algorithms/base.py

```diff
@@ -52,8 +52,8 @@ def __init__(
                 "Additionally, even unbiased algorithms can exploit "
                 "the information leak from the termination condition, "
                 "producing spuriously high performance. See "
-                "https://imitation.readthedocs.io/en/latest/guide/variable_horizon.html"
-                " for more information.",
+                "https://imitation.readthedocs.io/en/latest/getting-started/"
```

Comment:
I would honestly add an ignore to flake8, force splitting the URL feels like worse style to me than having a slightly longer line in this instance. But happy to see what others think.